Userland/Libraries/LibWeb/DOM/DOMImplementation.cpp

```diff
@@ -48,8 +48,8 @@ void DOMImplementation::visit_edges(Cell::Visitor& visitor)
 // https://dom.spec.whatwg.org/#dom-domimplementation-createdocument
 WebIDL::ExceptionOr<JS::NonnullGCPtr<Document>> DOMImplementation::create_document(Optional<String> const& namespace_, String const& qualified_name, JS::GCPtr<DocumentType> doctype) const
 {
-    // FIXME: 1. Let document be a new XMLDocument
-    auto xml_document = Document::create(realm());
+    // 1. Let document be a new XMLDocument
+    auto xml_document = XMLDocument::create(realm());
```

Comment:
Typo in commit message: this is createDocument, not createElement :^)

Reply:
derp, thanks!